client/boot/app.js

```diff
@@ -23,34 +23,33 @@ import {
 } from './common';
 import createReduxStoreFromPersistedInitialState from 'state/initial-state';
 import detectHistoryNavigation from 'lib/detect-history-navigation';
-import userFactory from 'lib/user';
+import { getUserFromCache, subscribeToUserChanges } from './user';
 
 const debug = debugFactory( 'calypso' );
 
-const boot = currentUser => {
+const bootProject = ( currentUser, reduxStore ) => {
 	debug( "Starting Calypso. Let's do this." );
 
+	configureReduxStore( currentUser, reduxStore );
+
 	const project = require( `./project/${ PROJECT_NAME }` );
 
-	locales( currentUser );
-	invoke( project, 'locales', currentUser );
+	locales( currentUser, reduxStore );
+	invoke( project, 'locales', currentUser, reduxStore );
```

Comment:
What's the benefit of `invoke` in that case instead of `project.locales( currentUser, reduxStore );` ?